build.sbt

```diff
@@ -68,22 +69,16 @@ libraryDependencies ++= Seq(
   geotrellisSpark,
   geotrellisSparkTestKit % Test,
   geotrellisS3,
-  geotrellisShapefile,
-  geotrellisGeotools,
-  geotrellisVectorTile,
   geotrellisGdal,
-  logging,
-  decline,
   sedonaCore,
   sedonaSQL,
-  //  jtsCore,
-  //  jts2geojson,
-  geoToolsOGRBridj,
-  bridj,
   breeze,
   breezeNatives,
   breezeViz,
   sparkDaria,
+  jts,
+  "org.datasyslab" % "geotools-wrapper" % "geotools-24.1",
+  "org.wololo" % "jts2geojson" % "0.14.3"
```

Comment:
These are odd, I was basically forced to include them by run-time exceptions, wololo I guess.